ProgramTask5.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        double y = F(2, 3, 1) + F(3, 8, 5) +F(5, 6, 1);
+        Console.WriteLine("x = " + Math.Round(y,3));
+
+        Console.ReadKey(); 
+    }
+
+static double F(double a, double b, double c)
+{
+    return Math.Sqrt((c+Math.Pow(Math.Tan(a),2))/b);
+}

```

Comment:
Все верно, но в методе можно было обойтись двумя параметрами
```suggestion
    static void Main(string[] args)
    {
        double y = F(1, 2) + F(5, 3) +F(1, 5);
        Console.WriteLine("x = " + Math.Round(y,3));

        Console.ReadKey(); 
    }

static double F(double a, double b)
{
    var tan = Math.Tan(b);
    return Math.Sqrt((a + tan * tan)/(a+b));
}

```